app/index.tsx

```diff
@@ -35,6 +35,7 @@ import { initStore } from './lib/store/auxStore';
 import { TSupportedThemes, ThemeContext } from './theme';
 import ChangePasscodeView from './views/ChangePasscodeView';
 import ScreenLockedView from './views/ScreenLockedView';
+import { RowHeightProvider } from './containers/RoomItem/useRowHeight';
```

Comment:
Root shouldn't import from a specific component like that.
Following colocation, you should place it on `lib/hooks`